File: lib/src/main/java/net/rafaeltoledo/databindingbug/lib/SampleActivity.java
package net.rafaeltoledo.databindingbug.lib;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;
import net.rafaeltoledo.databindingbug.lib.databinding.SampleBinding;

public class SampleActivity extends AppCompatActivity {

    private SampleBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.sample);
    }
}

File: lib/src/main/java/net/rafaeltoledo/databindingbug/lib/SampleActivity.kt
package net.rafaeltoledo.databindingbug.lib

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import net.rafaeltoledo.databindingbug.lib.databinding.SampleBinding

class SampleActivity : AppCompatActivity() {

    private lateinit var binding: SampleBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.sample)
    }
}